<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZonesInterfaces</class>
 <widget class="QMainWindow" name="ZonesInterfaces">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="interfacesAvailable">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>201</width>
      <height>511</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>201</width>
      <height>511</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>201</width>
      <height>511</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="dragAllRight">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dragOneRight">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dragOneLeft">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dragAllLeft">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="interfacesSelected">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>201</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="commit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Commit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>618</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
